components/UpdateProfile/UpdateProfile.jsx

```diff
@@ -96,13 +83,7 @@ export default function UpdateProfile({
         avatar_url: avatarUrl,
         avatar_path: newImagePath ? newImagePath : imagePath,
         slug: sluggedName,
-        sites: {
-          bandcamp: bandcampUrl ? prependProtocol(bandcampUrl) : null,
-          apple: appleMusicUrl ? prependProtocol(appleMusicUrl) : null,
-          spotify: spotifyUrl ? prependProtocol(spotifyUrl) : null,
-          soundcloud: soundcloudUrl ? prependProtocol(soundcloudUrl) : null,
-          youtube: youtubeUrl ? prependProtocol(youtubeUrl) : null,
-        },
+        sites: sites,
```

Comment:
I think you could run your sites through `prependProtocol` instead of doing it on every single input value change:

```jsx
sites: sites.map(site => prependProtocol(site))
```